packages/dynamite/dynamite/lib/src/models/openapi/parameter.dart

```diff
@@ -34,7 +61,7 @@ abstract class Parameter implements Built<Parameter, ParameterBuilder> {
       throw OpenAPISpecError('Path parameters must be required but ${b.name} is not.');
     }
 
```

Comment:
Please throw an `OpenAPISpecError` if both content and schema are specified.
According to the docs they are mutually exclusive.

Reply:
Wanted to add it - forgot to do so in the end :)